Sub/Src/Dynamics/movement_pid.py

```diff
@@ -218,12 +218,13 @@ def controlled_thrust(self, roll_control, pitch_control, yaw_control, x_control,
         for thruster_id, thruster in enumerate(self.thrusters):
             #Multiplied roll control by negative one to account for the thrusters going in the
             #wrong direction.
+
+            #(yaw_control * thruster.orientation[0] * thruster.location[1]) + \
```

Comment:
Remove this odd comment and say way we limit the yaw to only using thrusters 2 and 6.